File: sample-apps/layer-python/function-numpy/lambda_function.py
import json
import numpy as np

def lambda_handler(event, context):
    
    x = np.arange(15, dtype=np.int64).reshape(3, 5)
    print(x)
    
    return {
        'statusCode': 200,
        'body': json.dumps('Hello from Lambda!')
    }

File: sample-apps/layer-ruby/function/lambda_function.rb
require 'json'
require 'tzinfo'

def lambda_handler(event:, context:)
    tz = TZInfo::Timezone.get('America/New_York')
    { statusCode: 200, body: tz.to_local(Time.utc(2018, 2, 1, 12, 30, 0)) }
end

